src/styles/global.css

```diff
@@ -2,4 +2,14 @@
 
 @import '@astrojs/starlight-tailwind';
 @import 'tailwindcss/theme.css' layer(theme);
-@import 'tailwindcss/utilities.css' layer(utilities);
\ No newline at end of file
+@import 'tailwindcss/utilities.css' layer(utilities);
+
+
+body {
```

Comment:
The poppins font is applied here. We might in the future chose to clean up and make the typography a bit neater. But that can be in a separate PR. this ensures that maintenance of the docs is easier as it applies the styles in the config rather in the index file.